corehq/apps/app_manager/static/app_manager/js/details/screen_config.js

```diff
@@ -1109,7 +1109,7 @@ hqDefine('app_manager/js/details/screen_config', function () {
                             containsFilterConfiguration: columnType === "short",
                             containsCaseListLookupConfiguration: (columnType === "short" && (hqImport('hqwebapp/js/toggles').toggleEnabled('CASE_LIST_LOOKUP') || hqImport('hqwebapp/js/toggles').toggleEnabled('BIOMETRIC_INTEGRATION'))),
                             // TODO: Check case_search_enabled_for_domain(), not toggle. FB 225343
-                            containsSearchConfiguration: (columnType === "short" && hqImport('hqwebapp/js/toggles').toggleEnabled('SYNC_SEARCH_CASE_CLAIM')),
+                            containsSearchConfiguration: (columnType === "short" && hqImport('hqwebapp/js/toggles').toggleEnabled('USH_CASE_CLAIM_UPDATES')),
```

Comment:
https://github.com/dimagi/commcare-hq/pull/29735#discussion_r634715474
makes me think this is the correct flag.

It was originally set in https://github.com/dimagi/commcare-hq/pull/11438/files

Reply:
I don't think this is right. This flag controls the saving of the entire case search model: https://github.com/dimagi/commcare-hq/blob/0c8d6d5f404beca1a92d29ea8d0f1ff4e3ee95cf/corehq/apps/app_manager/static/app_manager/js/details/screen_config.js#L996-L998

That model contains SYNC_SEARCH_CASE_CLAIM features, not just USH_CASE_CLAIM_UPDATES features.

Reply:
Thanks. Will revert that.

is the understanding that any domain that would have `USH_CASE_CLAIM_UPDATES` would also have `SYNC_SEARCH_CASE_CLAIM` for sure?
Because if that is not then I am quite confused.

Reply:
this was the latest commit so i simply removed it for cleaner history

Reply:
Yeah, any domain with `USH_CASE_CLAIM_UPDATES` turned on also has to have `SYNC_SEARCH_CASE_CLAIM` turned on.